### DISCUSSION TITLE
Gradle Managed Devices

### DISCUSSION THREAD

**Discussion by gxwilkerson33** [Banno/Gordon#121] [Q&A] [answered]
Is it possible to use gradle managed devices to run the tests with gordon? We currently use the managed devices for running our instrumented tests during our CI/CD pipeline. Thanks

**Comment by joshschriever** [comment, marked as answer]
No, there's currently not a way to integrate gradle managed devices with Gordon. I would love that, but I've looked into it and not figured out a way to do it.
